supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -126,32 +1119,202 @@
 
 *See also*:
 
+[discrete]
+[[webservices-heading]]
+==== Web Services (noun)
+*Description*: Use "Web Services" when referring to the xref:webservices[webservices] subsystem in titles and headings. Write as two words. Capitalize both words. See xref:webservices[webservices] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: Web services, web services, webservices
+
+*See also*: xref:webservices[webservices]
+
+[discrete]
+[[webservices]]
+==== webservices (noun)
+*Description*: The "webservices" subsystem is used to configure the Web services provider. Write in lower case as one word. See xref:webservices-heading[Web Services] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: web services, Web services, Web Services
+
+*See also*: xref:webservices-heading[Web Services]
+
+[discrete]
+[[weld]]
+==== weld (noun)
+*Description*: The "weld" subsystem is used to configure Contexts and Dependency Injection (CDI) functionality for JBoss EAP. Write in lower case. See xref:weld-heading[Weld] entry for the correct usage in headings.
+
+*Use it*: yes
+
+*Incorrect forms*: Weld
+
+*See also*: xref:weld-heading[Weld]
+
+[discrete]
+[[weld-heading]]
+==== Weld (noun)
+*Description*: Use "Weld" when referring to the xref:weld[weld] subsystem in titles and headings. Capitalize the word. See xref:weld[weld] entry for the correct usage in general text.
+
+*Use it*: yes
+
+*Incorrect forms*: weld
+
+*See also*: xref:weld[weld]
+
 [discrete]
 [[windows-server]]
 ==== Windows Server (noun)
-*Description*: This uppercase term is correct when referring to Microsoft’s Windows Server product or Windows-specific commands and scripts like `standalone.bat`. "Microsoft" does not precede the product name.
+*Description*: Use "Windows Server" to refer to the Microsoft Windows Server product or Windows-specific commands and scripts such as `standalone.bat`. Do not precede the product name with "Microsoft".
+
+*Use it*: yes
+
+*Incorrect forms*: Microsoft Windows Server, Microsoft Windows, Windows
+
+*See also*: xref:microsoft-windows[Microsoft Windows]
+
+// ***********************
+// Terms starting with 'X'
+// ***********************
+
+[discrete]
+[[xml]]
+==== XML (noun)
+*Description*: "XML" is an acceptable shortened form of "eXensible Markup Language". Write in uppercase.
 
 *Use it*: yes
 
-*Incorrect forms*: Microsoft Windows, Windows
+*Incorrect forms*: Xml, xml
 
 *See also*:
+
+[discrete]
+[[xp]]
+==== XP (noun)
+*Description*: "XP" is an acceptable shortened form of "Expansion Pack". Write in upper case.
+
+*Use it*: yes
+
+*Incorrect forms*: Xp, xp
+
+*See also*: xref:expansion-pack[Expansion Pack]
+
+[discrete]
+[[xts]]
+==== xts (noun)
+*Description*: The "xts" subsystem is used to configure settings for coordinating Web services in a transaction. Write in lower case. Write in lower case as one word. See xref:xts-heading[XTS] entry for the correct usage in headings.
```

Comment:
Repetition - get rid of the extra `Write in lower case`.

Reply:
Fixed